app/views/administrate/application/_collection.html.erb

```diff
@@ -18,13 +18,13 @@ to display a collection of resources in an HTML table.
 [1]: http://www.rubydoc.info/gems/administrate/Administrate/Page/Table
 %>
 
-<table class="collection-data">
+<table class="collection-data" aria-labelledby="page-title">
```

Comment:
Yeah, great point. A `caption` element represents the exact need here. The only reason I reached for `aria-labelledby` instead is because we are already outputting the title on the page and this allowed me to easily hook onto that.

If we added a `caption`, we’d render the title twice in the DOM and likely just visually hide the `caption`. Using the `caption` _instead_ of the page’s `h1` that we already have won’t work with the layout already in place…